code/modules/vehicles/vehicle.dm

```diff
@@ -28,8 +28,9 @@
 
 //BUCKLE HOOKS
 /obj/vehicle/unbuckle_mob(mob/living/buckled_mob,force = 0)
-	riding_datum.restore_position(buckled_mob)
-	. = ..()
+	if(riding_datum)
```

Comment:
Shouldn't these be mandatory for vehicles ? I suppose broken subtypes could use it.